server/test/testFaceRestIncident.cc

```diff
@@ -226,6 +226,57 @@ void test_postIncidentForUnknownEvent(void)
 	assertEqualJSONString(expectedResponse, arg.response);
 }
 
+void test_putIncidentComment(void)
+{
+	loadTestDBIncidents();
+	startFaceRest();
+
+	size_t index = 2;
+	IncidentInfo expectedIncident = testIncidentInfo[index];
+	expectedIncident.status = "IN PROGRESS";
+	expectedIncident.priority = "HIGH";
+	expectedIncident.assignee = "@Mnagakawa";
+	expectedIncident.doneRatio = 30;
+	string path(
+	  StringUtils::sprintf("/incident/%" FMT_UNIFIED_EVENT_ID,
+			       expectedIncident.unifiedEventId));
+	RequestArg arg(path);
+	arg.userId = findUserWith(OPPRVLG_GET_ALL_SERVER);
+	arg.request = "PUT";
+	incidentInfo2StringMap(expectedIncident, arg.parameters);
+	arg.parameters["comment"] = "Assigned to @Mnagakawa";
+
+	// check the response
+	getServerResponse(arg);
+	string expectedResponse(
```

Comment:
Initializing string style to be mixed in same test case is not good.
Use 

```cpp
string hoge = "piyo";
```

style instead.